variables.tf

```diff
@@ -98,3 +98,15 @@ variable "infrastructure_execution_count" {
   type = number
   default = 1
 }
+
+variable "controller_instances_count" {
+  description = "The number of ec2 instances for controller"
+  type = number
+  default = 1
+}
+variable "hub_instances_count" {
```

Comment:
Change to infrastructure_hub_count to align with existing variables

Reply:
complete